test/e2e/tests/bridge/bridge-test-utils.ts

```diff
@@ -62,35 +57,27 @@ export class BridgePage {
       css: '[data-testid="multichain-token-list-button"]',
       text: symbol,
     });
-
-    await this.driver.delay(2000);
-    assert.ok((await this.driver.getCurrentUrl()).includes('asset'));
+    await this.driver.waitForUrlContaining({
+      url: 'asset',
+      timeout: this.driver.timeout,
+    });
   };
 
   verifyPortfolioTab = async (expectedHandleCount: number) => {
-    await this.driver.delay(4000);
+    await this.driver.waitUntilXWindowHandles(expectedHandleCount);
     await this.driver.switchToWindowWithTitle('MetaMask Portfolio - Bridge');
-    assert.equal(
-      (await this.driver.getAllWindowHandles()).length,
-      IS_FIREFOX || !isManifestV3
-        ? expectedHandleCount
-        : expectedHandleCount + 1,
-    );
```

Comment:
this logic is not needed, we can use the `waitUntilXWindowHandles` which takes care of the windows in the different builds for us